src/views/BrowseView/BrowseView.tsx

```diff
@@ -46,12 +55,20 @@ const BrowseView: React.FC<RouteComponentProps> = () => {
     throw categoriesError
   }
 
+  const isAllCategorySelected = selectedCategoryId === 'All'
+  const featuredVideos = featuredVideosData?.featuredVideos.map((featuredVideo) => featuredVideo.video)
+  const hasFeaturedVideosError = featuredVideosError && !featuredVideosLoading
   return (
     <Container>
       <BackgroundPattern />
       <Header variant="hero" ref={headerRef}>
-        Browse
+        Videos
```

Comment:
Since the view name changed, let's update the routing URL and nav link in the navbar. Let's also use the icon from here: https://www.figma.com/file/rBjfm2tJ2Pr9M0UNTduU9v/Joystream-Atlas?node-id=3560%3A1372